src/components/VideoHero/VideoHero.style.ts

```diff
@@ -1,26 +1,22 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
-import { fluidRange } from 'polished'
 
 import { Button, IconButton, SkeletonLoader, Text } from '@/shared/components'
-import { breakpoints, colors, media, sizes } from '@/shared/theme'
+import { colors, media, sizes, typography } from '@/shared/theme'
 
 import { ChannelLink } from '../ChannelLink'
 
 const CONTENT_OVERLAP_MAP = {
-  SMALL: 25,
-  MEDIUM: 150,
-  LARGE: 200,
-  XLARGE: 400,
-  XXLARGE: 600,
+  SMALL: 24,
+  MEDIUM: 24,
+  LARGE: 24,
+  XLARGE: 24,
+  XXLARGE: 24,
 }
-const GRADIENT_OVERLAP = 50
-const GRADIENT_HEIGHT = 250
-const INFO_BOTTOM_MARGIN = 100
-const BUTTONS_HEIGHT_PX = '54px'
 
 export const Container = styled.section`
   position: relative;
+  max-height: 100vh;
```

Comment:
Why do we need that?

Reply:
It's so on widescreens the hero section doesn't grow way bigger than the screen height and the info about the video channel doesn't get outside the screen, i.e
![imagen](https://user-images.githubusercontent.com/10615263/129901543-9593267d-d313-4ce5-9256-e3225176855f.png)

vs 

![imagen](https://user-images.githubusercontent.com/10615263/129901663-80916c08-e256-4889-a958-97217ccd0eb4.png)